to_do_list_part_two.py

```diff
@@ -1,7 +1,16 @@
 def add_task(task_list):
-    add_your_task = input("Enter the task:    ")
-    task_list.append(add_your_task)
-    print(f"{add_your_task} has been added to the list.")
+    task = input("Enter the task: ")
+
+    priority = input("Enter the priority(high, medium, low): ")
```

Comment:
You can check whether priority is `high` or `medium` or `low` here.

In line 7, you only check whether priority is empty or not.